README.rst

```diff
@@ -74,6 +74,9 @@ Usage is very basic.
    # update stats for all cars, and returns the list of cars. This will take
    # 60+ seconds per car
    cars = page.update_cars()
+   
+   # Optionally you can call update_cars() with a vin of your choice to only update that car.
+   cars = page.update_cars('vin number')
```

Comment:
I think that instead of putting the special logic on the update_cars to take vin, probably just make vin optional to the MyChevy constructor. Then get_cars() will just filter out the list of cars that don't match the vin, and should throw an exception if no car was found. That will let update_cars() remain unchanged, as the loop will just be a single car element.

Reply:
Oh I like that idea better. I will look into making the change to the constructor. I will also try to create a unit test for it too and then submit a new pull request.